wallet-new/src/Cardano/Wallet/Kernel/Transactions.hs

```diff
@@ -217,7 +217,7 @@ newTransaction ActiveWallet{..} spendingPassword options accountId payees = runE
     -- to compute the createdAt timestamp for `TxMeta`
     txMetaCreatedAt_  <- liftIO $ Node.getCreationTimestamp (walletPassive ^. walletNode)
     -- partially applied, because we don`t know here which outputs are ours
-    partialMeta <- liftIO $ createNewMeta accountId txId txMetaCreatedAt_ inputs (toaOut <$> outputs) True spentInputCoins
+    partialMeta <- liftIO $ createNewMeta accountId txId txMetaCreatedAt_ inputs (_txOutputs . taTx $ txAux) True spentInputCoins
```

Comment:
@kderme I suspect this line will alas conflict with the changes I did as part of CBR-440, sob.

Will give you a ping as soon as that gets merged to give you a clean slate.

Reply:
Ok, merged (and conflicts arose, heh).

Reply:
I hope all rebases where as easy as this one :)